tests/core/operation/test_export_analysis_operation.py

```diff
@@ -0,0 +1,127 @@
+import os
+
+import botocore
+from botocore.config import Config
+from botocore.stub import Stubber
+
+from core.operation.export_analysis_operation import ExportAnalysisOperation
+from tests.core.operation.analysis_test_responses import (
+    create_template_response,
+    describe_data_set_1_response,
+    describe_data_set_2_response,
+    describe_template_definition_response,
+    get_analysis_definition_response,
+    get_analysis_description_response,
+)
+
+
+class TestExportAnalysisOperation:
+    def test(self):
+        analysis_id = "my-quicksight-analysis-id"
+        output_dir = "/tmp/test-output"
```

Comment:
A more platform independent API exists under https://docs.python.org/3/library/tempfile.html
Or you can use the mocking strategy https://docs.python.org/3/library/unittest.mock.html#mock-open